source/patterns/02-organisms/content-header/content-header.twig

```diff
@@ -16,6 +16,15 @@
       {%- endblock categories -%}
     {%- endwith -%}
 
+    <a href="" class="content-header__download_link">
+      <picture>
+        <source srcset="/images/download-full.svg" type="image/svg+xml" media="(min-width: 45.625em)">
+        <source srcset="/images/download-full-2x.png" type="image/png" media="(min-width: 45.625em)">
```

Comment:
These media statements trigger at the threshold defined by the `medium` breakpoint (730px). This is shared data between Sass and the templating layer, which is the first occurrence of this type of data sharing. Will need to extend the config management to handle this (see [issue #151](https://github.com/libero/libero/issues/151)).